pom.xml

```diff
@@ -107,6 +107,22 @@
         <dependency>
             <groupId>io.jenkins.plugins</groupId>
             <artifactId>gson-api</artifactId>
-          </dependency>
+        </dependency>
+        <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>variant</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>ssh-credentials</artifactId>
+          <version>340.vf0948074a_a_23</version>
```

Comment:
:warning: incremental version

Reply:
I am gonna drop that commit. Was just a test that it in fact works.